src/ActionModal/components/DelegationModal.vue

```diff
@@ -186,6 +190,12 @@ export default {
       this.selectedIndex = 0
       this.amount = null
     },
+    passToParent(func) {
+      this.validateChangeStep = func
+    },
+    enterPressed() {
+      this.validateChangeStep()
```

Comment:
rather then passing functions around, you can reference the actionModal
```suggestion
      this.$refs.actionModal.validateChangeStep()
```
I recommend that you get rid of the `passToParent` to clean up some code.

Reply:
Much simpler. I will update